cmd/main.go

```diff
@@ -343,6 +350,12 @@ func launchNodeWithConfig(c *cli.Context) error {
 		natAddr = fmt.Sprintf("%s:%d", c.String(optionNATAddr.Name), c.Int(optionNATPort.Name))
 	}
 
+	crtFile := c.String(serverTLSCertificateFlagName)
+	keyFile := c.String(serverTLSPrivateKeyFlagName)
+	if crtFile != "" && keyFile == "" || crtFile == "" && keyFile != "" {
```

Comment:
Could we do it in a simpler way?
```suggestion
	if (crtFile == "") != (keyFile == "") {
```